src/app/frontend/deploy/helpsection/userhelp.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.kd-user-help * {
+  opacity: 0.5;
+}
+
+.kd-user-help a {
+  color: #326de6;
+  opacity: 1;
+  text-decoration: none;
+}
+
+.kd-user-help a:hover {
+  color: #1254df;
```

Comment:
This looks like application wide color. Can you add it to the SCSS file with other color constants ?